addon/components/frost-button.js

```diff
@@ -69,11 +74,22 @@ export default Component.extend(PropTypeMixin, {
     text: PropTypes.string,
     title: PropTypes.string,
     type: PropTypes.string,
-    vertical: PropTypes.bool
+    vertical: PropTypes.bool,
+
+    // state
```

Comment:
I was originally asking to remove empty sections, but as long as the number of sections remains reasonably low, I'm fine with keeping them.

I want to avoid a situation where I have to scroll through reams of empty comments to see meaningful code (I've been in cases before with 80+ lines of empty comments per file).  I think we're avoiding that with the single line section headers and the low number of sections we have.